### PR TITLE
BD-3286 Catalog use cases and explanation

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

Add use cases to catalogs and an explanation about how it relates/differs to Liquid

### Related PRs, issues, or features (optional)

#BD-3286

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs.md [hunk 1]
```diff
@@ -28,3 +28,33 @@ guide_featured_list:
...
+- **Media preferences:** Import information about movies and shows, then recommend content to your users based on their favorited titles and most-watched genres.
+
+### Travel and hospitality
+
+- **Destinations:** Import travel destinations and their most popular attractions, restaurants, and activities, then personalize recommendations to your users based on their previous trips.
+- **Accomodations:** Import hotel properties and their amenities, room types, and pricing, then send promotions to your users based on their selected preferences.
+- **Travel methods**: Import deals and promotions for travel modes (such as flights, trains, rental cars, and others), then send them to your users based on their recent search history.
+- **Meal preferences:** Import information about meal offerings and use [selections]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/selections/) to send personalized messages to users who have specific meal preferences based on their most recently viewed food category.
+
+## How do catalogs and Liquid relate to each other?
```
Comment (rachel-feinberg) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs.md:56]:
```suggestion
## How catalogs and Liquid work together
```

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs.md [hunk 2]
```diff
@@ -28,3 +28,33 @@ guide_featured_list:
...
+
+Catalogs are a data storage feature. They contain large sets of data that can be referenced in your messages for personalization. To actually reference the data, you'll use Liquid as the templating language. In other words, catalogs is storage where the data is held, and Liquid is the language that pulls the relevent data from the storage.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs.md:58]:
```suggestion
Catalogs are a data storage feature. They contain large sets of data that can be referenced in your messages for personalization. To actually reference the data, you'll use [Liquid ]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/) as the templating language. In other words, catalogs is storage where the data is held, and Liquid is the language that pulls the relevent data from the storage.
```

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs.md [hunk 3]
```diff
@@ -28,3 +28,33 @@ guide_featured_list:
   image: /assets/img/braze_icons/server-01.svg
 ---
 <br><br>
+
+## Catalog use cases
+
+You can bring in any type of data into a catalog. Typically, the data is metadata about offerings, such as products, discounts, promotions, events, and similar. See the use cases below for a few examples of how you can use this data to target users with highly-relevent messaging.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs.md:34]:
```suggestion
You can bring in any type of data into a catalog. Typically, the data is metadata about offerings, such as products, discounts, promotions, events, and similar. See the use cases below for a few examples of how you can use this data to target users with highly relevant messaging.
```

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs.md [hunk 4]
```diff
@@ -28,3 +28,33 @@ guide_featured_list:
...
+- **Accomodations:** Import hotel properties and their amenities, room types, and pricing, then send promotions to your users based on their selected preferences.
+- **Travel methods**: Import deals and promotions for travel modes (such as flights, trains, rental cars, and others), then send them to your users based on their recent search history.
+- **Meal preferences:** Import information about meal offerings and use [selections]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/selections/) to send personalized messages to users who have specific meal preferences based on their most recently viewed food category.
+
+## How catalogs and Liquid work together
+
+Catalogs are a data storage feature. They contain large sets of data that can be referenced in your messages for personalization. To actually reference the data, you'll use [Liquid]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/) as the templating language. In other words, catalogs is storage where the data is held, and Liquid is the language that pulls the relevent data from the storage.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs.md:58]: Love this!

**Review by bre-fitzgerald — APPROVED** [review]
Just realized I forgot to approve 😆 Lgtm! As a future thought, for the section "How catalogs and Liquid work together", we might want to consider creating a simple graphic to illustrate that relationship for visual learners.

**Pull request merged**